modules/intermodal/src/intermodal.cc

```diff
@@ -38,6 +38,7 @@ namespace motis::intermodal {
 intermodal::intermodal() : module("Intermodal Options", "intermodal") {
   param(router_, "router", "routing module");
   param(revise_, "revise", "revise connections");
+  param(timeout_, "timeout", "timeout in seconds for searching for routes (0 = no timeout)");
```

Comment:
maybe more compact comment "routing timeout in seconds (0 = no timeout)"
```suggestion
  param(timeout_, "timeout", "routing timeout in seconds (0 = no timeout)");
```